src/app/shared/components/managecitations/managecitations.component.html

```diff
@@ -50,87 +50,148 @@
-							</div>
-							<div>
-								<span *ngIf="!c.isEditing">{{ c.title }}</span>
-								<span *ngIf="c.isEditing"><textarea class='break' name="title" [(ngModel)]="c.title"
-										required></textarea></span>
-							</div>
-							<div>
-								<a *ngIf="!c.isEditing" [href]="c.citationURL" [target]="_blank">
-									<span *ngIf="!c.isEditing">{{ c.citationURL }}</span>
-								</a>
-								<span *ngIf="c.isEditing"><textarea class='break' name="citationURL"
-										[(ngModel)]="c.citationURL" required></textarea></span>
-							</div>
-						</td>
-						<!-- Col 2 - Citation Buttons -->
-						<td class=editCol>
-							<!-- Editing -->
-							<div [class.hidden]="!showAddCitations" class="cell-buttons">
-								<button *ngIf="!c.isEditing" title="Edit Citation" (click)="editRowClicked(c, cIndex)"
-									class="icon-button" [disabled]="!checkManagerCitations(c)" aria-hidden="true">
-									<i class="far fa-pencil"></i>
-								</button>
-								<button *ngIf="c.isEditing" title="Save Citation" (click)="saveCitation(c)"
-									class="icon-button" aria-hidden="true">
-									<i class="far fa-floppy-o"></i>
-								</button>
-								<button *ngIf="c.isEditing" title="Cancel Edit" (click)="CancelEditRowClicked()"
-									class="icon-button" aria-hidden="true">
-									<i class="far fa-history"></i>
-								</button>
-								<button *ngIf="loggedInRole === 'Administrator' && !c.isEditing" title="Delete Citation"
-									(click)="deleteCitation(c.id)" class="icon-button"
-									[disabled]="c.regressionRegions?.length>0" aria-hidden="true">
-									<i class="far fa-times"></i>
-								</button>
-							</div>
-							<!-- Not editing -->
-							<div [class.hidden]="showAddCitations" class="cell-buttons">
-								<button title="Add Citation" (click)="addExistingCitation(c); d('closed')"
-									class="icon-button" aria-hidden="true">
-									<i class="far fa-plus"></i>
-								</button>
-							</div>
-						</td>
+						<!-- When the Citation Modal is not opened in the GagePage -->
+						<ng-container *ngIf="!inGagePage">	
+							<!-- Col 1, Citation -->
+							<td class=col1>
+								<div class="half-space">
+									<span *ngIf="!c.isEditing">{{ c.author }}</span>
+									<span *ngIf="c.isEditing"><textarea class='break' name="author" [(ngModel)]="c.author"
+											required></textarea></span>
+								</div>
+								<div>
+									<span *ngIf="!c.isEditing">{{ c.title }}</span>
+									<span *ngIf="c.isEditing"><textarea class='break' name="title" [(ngModel)]="c.title"
+											required></textarea></span>
+								</div>
+								<div>
+									<a *ngIf="!c.isEditing" [href]="c.citationURL" [target]="_blank">
+										<span *ngIf="!c.isEditing">{{ c.citationURL }}</span>
+									</a>
+									<span *ngIf="c.isEditing"><textarea class='break' name="citationURL"
+											[(ngModel)]="c.citationURL" required></textarea></span>
+								</div>
```

Comment:
Again, this section and on has a lot of repeats that can be treated more or less the same if it's opened from gage page or not.